cdk/src/open-data-platform/data-plane/data-import/write-water-systems-data.handler.ts

```diff
@@ -58,29 +58,49 @@ function getValueOrDefault(field: string): number {
  * Maps a data row to a table row ready to write to the db.
  * @param row: row with all data needed to build a [WaterSystemsTableRow].
  */
-function getTableRowFromRow(row: any): SqlParametersList {
+function getTableRowFromRow(row: any): SqlParametersList | null {
   const value = row.value;
   const properties = value.properties;
-  return (
-    new WaterSystemsTableRowBuilder()
-      .pwsId(properties.pwsid)
-      .pwsName(properties.pws_name ?? '')
-      .leadConnectionsCount(getValueOrDefault(properties.lead_connections))
-      .serviceConnectionsCount(getValueOrDefault(properties.service_connections_count))
-      .populationServed(getValueOrDefault(properties.population_served_count))
-      // Keep JSON formatting. Post-GIS helpers depend on this.
-      .geom(JSON.stringify(value.geometry))
-      .build()
-  );
+
+  console.log(`properties: ${JSON.stringify(properties)}`);
+
+  if (value.geometry != null) {
+    const reportedLeadConnections = getValueOrDefault(properties.lead_connections);
+    let leadConnectionsCount;
+
+    // If there is a reported lead count, use that
+    if (reportedLeadConnections > 0) {
+      leadConnectionsCount = reportedLeadConnections;
+      // Otherwise, get estimate from prediction interval.
+    } else {
+      leadConnectionsCount =
+        (getValueOrDefault(properties.low) + getValueOrDefault(properties.high)) / 2;
```

Comment:
why store an avg instead of low/high?

Reply:
Added a TODO but basically storing these values would require changing a bunch of tables so we need to do this separately